ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -216,6 +216,17 @@ const intermediateRelativityOutputStrings = {
   },
 };
 
+const directions = {
```

Comment:
These keys show up in the config ui.  Would you mind adding a layer of indirection here?

In other words, add a map that's `const directions = { 0: output.north(), 1: output.northeast()`, etc}` to the infotext itself, and then add the directions with their real names, e.g. `north: Outputs.north` and so forth to the output strings.